source/Mortgage.cpp

```diff
@@ -27,17 +27,17 @@ using namespace std;
 
 // Find out how much you can afford to borrow with the given annual revenue
 // and the given credit score (which should be between 200 and 800).
-int64_t Mortgage::Maximum(int64_t annualRevenue, int creditScore, int64_t currentPayments)
+int64_t Mortgage::Maximum(int64_t annualRevenue, int creditScore, double currentPayments)
 {
 	const int term = 365;
-	annualRevenue -= term * currentPayments;
-	if(annualRevenue <= 0)
+	double revenue = annualRevenue - term * currentPayments;
```

Comment:
```suggestion
	const double revenue = annualRevenue - term * currentPayments;
```
I don't see it getting changed.